serializers/src/glTF/2.0/babylon.glTFExporter.ts

```diff
@@ -186,121 +216,39 @@ module BABYLON.GLTF2 {
          * @param strideSize - Represents the offset between consecutive attributes
          * @param byteOffset - The offset to start counting bytes from.
          * @param dataBuffer - The buffer to write the binary data to.
-         * @param useRightHandedSystem - Indicates whether the data should be modified for a right or left handed coordinate system. 
          * @returns - Byte length of the attribute data.
          */
-        private writeAttributeData(vertexBufferKind: string, meshAttributeArray: FloatArray, strideSize: number, vertexBufferOffset: number, byteOffset: number, dataBuffer: DataView, useRightHandedSystem: boolean): number {
+        private writeAttributeData(vertexBufferKind: string, meshAttributeArray: FloatArray, strideSize: number, vertexBufferOffset: number, byteOffset: number, dataBuffer: DataView): number {
             let byteOff = byteOffset;
 
-            const start = 0;
             const end = meshAttributeArray.length / strideSize;
 
             let byteLength = 0;
 
-            switch (vertexBufferKind) {
-                case VertexBuffer.PositionKind: {
-                    for (let k = start; k < end; ++k) {
-                        const index = k * strideSize!;
-                        dataBuffer!.setFloat32(byteOff, meshAttributeArray![index], true);
-                        byteOff += 4;
-
-                        dataBuffer!.setFloat32(byteOff, meshAttributeArray![index + 1], true);
-                        byteOff += 4;
-
-                        if (useRightHandedSystem) {
-                            dataBuffer!.setFloat32(byteOff, meshAttributeArray![index + 2], true);
-                        }
-                        else {
-                            dataBuffer!.setFloat32(byteOff, -meshAttributeArray![index + 2], true);
-                        }
+            for (let k = 0; k < end; ++k) {
+                const index = k * strideSize;
+                dataBuffer!.setFloat32(byteOff, meshAttributeArray[index], true);

```

Comment:
! isn't necessary

Reply:
Thanks, I caught that in a previous revision.  There should no longer be any in the glTFExporter.ts file.